uniswap_swap_bot/src/bot.spec.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  createTransactionEvent,
+} from "forta-agent";
+import { createAddress } from "forta-agent-tools";
+
+import bot from "./bot";
+import { UNISWAP_ADDRESS, SWAP_EVENT } from "./constant";
+
+describe("swap occur", () => {
```

Comment:
Another standard the team follows is that the `describe` string should be something like `"UniswapV3 Swap Detector Test Suite"`, as an example.